packages/auth/src/grant/routes.test.ts

```diff
@@ -265,6 +268,29 @@ describe('Grant Routes', (): void => {
           error: 'interaction_required'
         })
       })
+
+      test('Fails to initiate a grant if payment pointer has no public name', async (): Promise<void> => {
+        jest.spyOn(clientService, 'get').mockReturnValueOnce(undefined)
```

Comment:
Can this mock a payment pointer with an undefined `publicName`?

Reply:
I don't think so 
`Property 'name' is missing in type '{ uri: string; }' but required in type 'ClientDetails'.`